### PR TITLE
Move appdata into new metainfo file location: see:

### PR DESCRIPTION
**Pull request by LocutusOfBorg**
https://wiki.debian.org/AppStream/Guidelines

I also changed the license to GPL-2, so it is in sync with the rest of the code.

### CODE REVIEW

**Comment by LocutusOfBorg** [comment]
I would like to merge this one on the next few days if nobody objects!

File: desktop/ettercap.metainfo.xml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+            fingerprints (active or passive) to find the geometry of the LAN.
+    </p>
+  </description>
+  <releases>
+    <release version="0.8.3"/>
+  </releases>
+
+  <screenshots>
+    <screenshot height="600" width="800" type="default">
+      <image>http://ettercap.github.io/ettercap/screenshots/Screenshot-1.png</image>
```
Comment (koeppea) [inline desktop/ettercap.metainfo.xml]: The URL should be https://www.ettercap-project.org/screenshots/Screenshot-1.png. The redirect behind ettercap.github.io redirects unconditionally.

File: desktop/ettercap.metainfo.xml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      <image>http://ettercap.github.io/ettercap/screenshots/Screenshot-2.png</image>
```
Comment (koeppea) [inline desktop/ettercap.metainfo.xml]: Same here.

**Comment by LocutusOfBorg** [comment]
Fixed, rebased and pushed :)

**Comment by koeppea** [comment]
Thanks merging..

**Pull request merged**